appeals/web/src/server/views/appeals/appeal/reject-ip-comment.njk

```diff
@@ -28,21 +25,27 @@
 			{%- set addAnotherReasonHtml = addAnotherReason({
 				textItems: option.textItems,
 				inputName: 'rejectionReason-' + option.value,
+				inputId: 'rejection-reason-' + option.value,
         labelText: 'Enter a reason',
         error: option.error
 			}) -%}
 			{%- set _ = option | addConditionalHtml(addAnotherReasonHtml) -%}
 		{%- endif -%}
 	{%- endfor -%}
 	<form method="post" novalidate="novalidate">
-		<div class="govuk-fieldset">
```

Comment:
How come we're dropping this fieldset? Does it throw the style off or something?